api/client/types.ts

```diff
@@ -122,31 +133,40 @@ export interface ErrorResponse extends AtatResponse {
   errorMessage?: string;
   errorCode?: string;
 }
+
 export interface AddPortfolioRequest extends ProvisionRequest {
   readonly portfolio: Portfolio;
 }
 export interface AddPortfolioResponseSync extends AtatResponse {
   readonly portfolio: Portfolio;
 }
-export type AddPortfolioResponseAsync = AsyncProvisionResponse;
+export interface AddEnvironmentRequest extends ProvisionRequest {
+  readonly portfolioId: string;
+  readonly environment: Environment;
+}
+export interface AddEnvironmentResponseSync extends AtatResponse {
+  readonly environment: Environment;
+}
+export type AddEnvironmentResponseAsync = AsyncProvisionResponse;
 
-export interface GetPortfolioRequest extends AtatRequest {
+export interface GetPortfolioRequest extends ProvisionRequest {
   readonly portfolioId: string;
 }
 export interface GetPortfolioResponse extends AtatResponse {
-  readonly portfolio: Portfolio;
+  readonly portfolio: Environment;
 }
 
-export interface PatchPortfolioRequest extends ProvisionRequest {
+export interface PatchEnvironmentRequest extends ProvisionRequest {
   readonly portfolioId: string;
-  readonly patch: PortfolioPatchInput;
+  readonly environmentId: string;
+  readonly patch: EnvironmentPatchInput;
 }
-export interface PatchPortfolioResponseSync extends AtatResponse {
-  readonly patch: PortfolioPatchInput;
+export interface PatchEnvironmentResponseSync extends AtatResponse {
+  readonly patch: EnvironmentPatchInput;
 }
-export type PatchPortfolioResponseAsync = AsyncProvisionResponse;
+export type PatchEnvironmentResponseAsync = AsyncProvisionResponse;
 
-export interface GetCostsByPortfolioRequest extends AtatRequest {
+export interface GetCostsByPortfolioRequest extends ProvisionRequest {
   readonly portfolioId: string;
   readonly startDate: string;
   readonly endDate: string;
```

Comment:
No, I should have left a comment here to indicate why I did this. Basically, when I removed the target impact level header, it made `AtatRequest` a meaningless interface because it had no properties (literally an error in TypeScript). So the quick and dirty fix was to just pick a new one - otherwise I'd have to have gone into the client code and made more changes, which I did not want to do. I'll go back and add a TODO for this though.